tests/gcp/unit_tests/workload_identity/test_common_provider_owner.py

```diff
@@ -0,0 +1,26 @@
+"""
+Terraform external provider just handles strings in maps, so tests need to consider this
+"""
+from sys import path, stderr
+
+try:
+    path.insert(1, '../../../test_fixtures/python_validator')
+    from python_validator import python_validator
+except Exception as e:
+    print(e, stderr)
+
+
+"""
+    Tests that an owner attribute defined in the components.common block, is set in all providers except those 
+    where that value is overwritten 
+"""
+
+expected_data = {
+    "bitbucket": "mesoform",
```

Comment:
If you are using collection values in the test make all the values something differ so that you can confirm something wasn't accidentally assigned to some other key

Reply:
This test is to test that the `components.common` block is can be used to assign a common `owner` value across the specified provider, by using 
```yaml
components:
  common:
    providers:
      owner: mesoform
```
The `gitlab`  one is the only one that is different because it was overwritten in the `components.specs.cicd.providers.gitlab` block. 
Having different values would not be clear that the `components.common` block is working as expected